pages/1.11/installing-upgrading/cloud/packet/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: layout.pug
+excerpt:
+title: Installing on Packet
+navigationTitle: Installing on Packet
+menuWeight: 20
+
+oss: true
```

Comment:
@masonse Can users follow these instructions to install Enterprise on Packet? They rely on terraform so maybe yes?